Add the pager name to busybox more to support use in minimal environments
which use apt and Debian packaging scripts

--- a/util-linux/more.c
+++ b/util-linux/more.c
@@ -24,6 +24,7 @@
 //config:	any need to reading text files, you can leave this disabled.
 
 //applet:IF_MORE(APPLET(more, BB_DIR_BIN, BB_SUID_DROP))
+//applet:IF_MORE(APPLET_ODDNAME(pager, more, BB_DIR_BIN, BB_SUID_DROP, more))
 
 //kbuild:lib-$(CONFIG_MORE) += more.o
 
